.cirrus/modules/build.star

```diff
@@ -40,7 +40,7 @@ def build_env():
     env |= next_env()
     env |= {
         "DEPLOY_PULL_REQUEST": "true",
-        "BUILD_ARGUMENTS": "--build-cache"
+        "BUILD_ARGUMENTS": "lintGoCode --build-cache"
```

Comment:
I guess is not needed as the task now is a dependency for `check` and `sonar` tasks

Reply:
Good catch